packages/neo-one-monitor/src/MonitorBase.js

```diff
@@ -780,6 +792,73 @@ export default class MonitorBase implements Span {
     }
   }
 
+  report(report: Report): void {
+    report.logs.forEach(log => {
+      const newLog = {
+        name: log.name,
+        level: log.level,
+        message: log.message,
+        labels: log.labels,
+        data: log.data,
+        error: undefined,
+      };
+      const { error } = log;
+      if (error != null) {
+        newLog.error = new Error(error.message);
+        if (error.stack != null) {
+          newLog.error.stack = error.stack;
+        }
+      }
+      this._logger.log(newLog);
+    });
+
+    for (const counterName of Object.keys(report.metrics.counters)) {
+      const counterMetric = report.metrics.counters[counterName];
+      const counter = this.getCounter({
+        name: counterMetric.metric.name,
+        help: counterMetric.metric.help,
+        labelNames: counterMetric.metric.labelNames,
+        directReport: true,
+      });
+
+      counterMetric.values.forEach(value => {
+        if (
+          value.countOrLabels != null &&
+          typeof value.countOrLabels === 'object'
+        ) {
+          counter.inc(value.countOrLabels, value.count);
```

Comment:
I think this should work without checks, i.e. just do `counter.inc(value.countOrLabels, value.count)` without any checks.